### ISSUE TITLE
Displayed transfer speed does not reflect the real transfer speed

### ISSUE THREAD

**Issue by soupdiver** [restic/restic#1581]
I started a backup yesterday and resumed it today. After scanning it seems restic verifies which file are already back-upped. It shows a huge transfer speed of 50Mb/s for these files. When it continues to actually transfer files the current speed is way too high. It shows 20Mb/s for me for example but my system is only sending data with 2Mb/s. The speed also gows down all the time. So I guess the (avg) speed is calculated over the whole transfer time and not the last x seconds for example.
It's just a small/cosmetic issue or maybe even intended. But I think it's not intuitive.

## Output of `restic version`
restic 0.8.1
compiled with go1.9.2 on linux/amd64

## How did you run restic exactly?
`restic -r sftp: ...` followed by repo path, name and local folder to backup `~` in my case.

## What backend/server/service did you use to store the repository?
sftp

## Expected behavior
I expect to see the _real_ transfer speed.
## Actual behavior
I see an inaccurate transfer speed.

## Steps to reproduce the behavior
Resume a backup

## Do you have any idea what may have caused this?
The speed is probably calculated over the whole transfer time instead let's say the last 5s.

## Do you have an idea how to solve the issue?
Calculate the transfer speed based on a smaller time windows.

## Did restic help you or made you happy in any way?
nope

**Comment by fd0** [comment]
You're right, that's not intuitive. We'll probably remove the "data processed bandwidth" eventually and replace it with a "backend read/write bandwidth" indicator.

> Did restic help you or made you happy in any way?
> nope

:cry:

**Comment by fd0** [comment]
This issue is connected to #1033, but I think it's not a duplicate.

**Comment by MichaelEischer** [comment]
As requested in #1850 by @alphapapa:
> It would also be nice if it could display the "overhead," i.e. how much data is sent and received from the remote backend that is not actual backed-up data (e.g. Obnam has a nice summary of overhead at the end of a backup run). It might not be useful to do show the overhead for local repositories.

**Comment by MichaelEischer** [comment]
@fd0 mentioned in #2131 a possible way to implement this:
> Since we already have bandwidth limiting in place we can use the same hook to monitor bandwidth to/from the backend and show it e.g. during backup.

**Comment by solarkraft** [comment]
Monitoring processed files instead of upload speed also significantly messes with the ETA. In my case it starts in the seconds and just keeps ballooning, which isn’t that nice.
